docs/library/urandom.rst

```diff
@@ -0,0 +1,58 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator.
+
+Given below are the methods that this module provides.
+
+ 
+  .. method:: urandom.getrandbits(n)
+
+    Returns an integer with 'n' random bits (1<=n<=32).
+    
+  .. method:: urandom.choice(sequence)
+   
+    Chooses multiple random items from a sequence (list, set or any
+    data structure).
+    
+  .. method:: urandom.seed(n=None, /)
+  
+    Initialises random number generator module with n. The None case only 
+    works if :MICROPY_PY_URANDOM_SEED_INIT_FUNC: is defined in the port, 
+    otherwise raises ValueError.
```

Comment:
When no argument (or None) is passed in it will (if supported by the port) initialise the PRNG with a true random number (usually a hardware generated random number).